### ISSUE TITLE
Avoid targeting YML files in "Learn more" links

### ISSUE THREAD

**Issue by tidoust** [web-platform-dx/baseline-status#17]
Reviewing the proposal to add a quick lookup to the Baseline landing page in https://github.com/web-platform-dx/web-features/pull/1800 makes me realize that the "Learn more" link is a bit abrupt: it takes me to a YML data file when I would rather have expected to, well, learn more about the feature in a more digestable way.

Since the group also maintains a Web Features explorer, one idea could be to link to the page that describes the feature in the explorer, for example, for `appearance`, link could be to https://web-platform-dx.github.io/web-features-explorer/features/appearance/ instead of to https://github.com/web-platform-dx/web-features/blob/main/features/appearance.yml

An alternative could be to link to a relevant page on MDN, but not all features have MDN pages.

(cc @captainbrosset)

**Comment by atopal** [comment]
Yes, that was not the intention. The intention to give more info about Baseline and now that we have a landing page for it, it would link to the landing page.

**Comment by captainbrosset** [comment]
Linking to the baseline landing page seems like the right default thing to do. I would also suggest making the link customizable via the component API surface. While existing consumers want to link to information about baseline, other future consumers might want to link to places to learn more about that particular feature.

**Comment by voxpelli** [comment]
If one could link the `<h1>` (or if its changed as asked in #28) as well that would be nice – my initial reaction was to try clicking the title and thinking I would end up at the MDN page for the feature, then when I found out it wasn't linked I clicked the "Learn more" and experienced this

**Comment by bramus** [comment]
We - Chrome CSS and UI DevRel - have rolled out the widget on https://chrome.dev/css-wrapped-2024/ and have received several reports from users that the "Learn More" link is totally useless. 

I can't say I disagree with them.

Short term solutions:
- Remove the link 
- Rename the link to "Data Source" or something like that
- Replace the link with "Learn more about Baseline" (but personally I would find that less useful)

Long term solutions:
- Update all entries so that they link to, for example, the MDN page on the topic.

**Comment by argyleink** [comment]
I wish it took me straight to caniuse. A good example here is `paint-order`

![Screenshot 2024-12-11 at 7 30 34 AM](https://github.com/user-attachments/assets/d1b8222e-da78-473e-8a27-c37b5ef65ef4)

it's all checks.. but says it's limited? _clicks expando details arrow_

![Screenshot 2024-12-11 at 7 30 25 AM](https://github.com/user-attachments/assets/47c5a649-09bb-4da2-9066-ba591fe49726)

but why? so _clicks learn more_…

![Screenshot 2024-12-11 at 7 30 47 AM](https://github.com/user-attachments/assets/5ad43ad6-c3f2-4a28-b3da-161d29c19e83)

this lack of transparency about the "limited" feature label hurts adoption

**Comment by captainbrosset** [comment]
That's a very good point Adam. Turns out the feature is [not supported on Safari for iOS](https://web-platform-dx.github.io/web-features-explorer/features/paint-order/), but the baseline status component doesn't show that!

I think we should take that particular point out and into its own separate issue (I'll create it soon).

@argyleink is your suggestion to link to caniuse something you'd want specifically for this one feature (because of the weird limited+green state)? Or something you think would be useful in all cases?

**Comment by devnook** [comment]
Since the widget relies on data from [webstatus.dev](https://webstatus.dev/), should we point "Learn more" link to the `https://webstatus.dev/features/<feature-id>` URL?

**Comment by bramus** [comment]
I don’t think linking to `https://webstatus.dev/features/<feature-id>` _(as it is right now)_ adds any value, because those pages are basically dead ends. From an author POV, the primary goal of a “Learn more” link should be to give them more info about the feature, not to point them to a chart that shows where the numbers were gotten from.

Take `anchor-positioning` for example. That https://webstatus.dev/features/anchor-positioning page does not give the user any info on what the feature is or how to use it. That page only offers info about how the numbers evolved over time and a link to CanIUse _(which isn’t present for all features)_.

Some solutions I can think of:

1. Include two links in the widget:
    - Have the “Learn more…” link link to MDN to give authors the info they are most likely looking for.
    - Add a second “Data Source” link that links to `webstatus.dev`.
2. Link to `webstatus.dev` but fix that site so that it actually has more info
    - Short description for the feature.
    - Details about which sub-features are part of the Baseline feature-set.
    - Details about browser versions and support.
    - Link to MDN.
    - etc.

**Comment by tunetheweb** [comment]
I think short term we should link to the dist link as that, while still an unfriendly YAML file, at least provides more info as to what is missing for each browser (in an admittedly kinda-cryptic manner until you get used to reading them).

I raised https://github.com/web-platform-dx/baseline-status/pull/75 for this.

Thoughts?

**Comment by rviscomi** [comment]
I agree with @tidoust that the YAML files aren't very digestible and linking to MDN would be ideal.  https://github.com/GoogleChrome/webstatus.dev/issues/930 is the only blocker. We could also link to the explorer, but we should name the links more descriptively than "Learn more".

**Comment by tunetheweb** [comment]
Some downsides of that is that include:
- some features would have many MDN links
- some BCD data doesn't have MDN links
- sometimes only SOME of the BCD data on a page may be relevant for that feature (especially for additions to an older feature), so figuring out which is used is complicated again...

If we were going to add MDN links, then I think we should have another link to dist YAML to more easily identify issues. Maybe under a better name than "Learn more" (e.g. "Web-Features Data" or such like).

**Comment by captainbrosset** [comment]
Putting myself in the shoes of a web developer reading https://chrome.dev/css-wrapped-2024/ (which was mentioned previously), being interested in a feature, and clicking **Learn more**, I 100% agree that ending up on the feature's YML file (or the dist YML file for that matter) is super confusing and not useful. As a developer, I don't want to learn more about how and where the feature is stored. I want to learn more about _the feature_ itself.

Even the dist file isn't very useful because it contains browser compat data which the widget already shows in a more visual way.

This widget is _for_ developers. So the **Learn more** link should go to a place where devs can learn more about the feature.

MDN would be ideal. It's the natural center of gravity for web feature reference docs. But because there isn't always a 1:1 mapping, it's a bit risky.

The web-features-explorer site is the next best thing in my opinion because:
* It's guaranteed that `https://web-platform-dx.github.io/web-features-explorer/features/<feature-id>` exists.
* It almost always has links to MDN pages.
* It's vendor-neutral.
* The site has more information about Baseline, should people want to learn more.

We could also fix this with a two steps process:
1. Link to MDN _directly_ when possible, by using https://github.com/web-platform-dx/web-features-mappings/blob/main/mappings/mdn-docs.json and using the mapping entries where 1 feature = 1 MDN link.
2. If not possible to link to MDN directly, link to the explorer.

**Comment by tunetheweb** [comment]
> Putting myself in the shoes of a web developer reading https://chrome.dev/css-wrapped-2024/ (which was mentioned previously), being interested in a feature, and clicking Learn more, I 100% agree that ending up on the feature's YML file (or the dist YML file for that matter) is super confusing and not useful. As a developer, I don't want to learn more about how and where the feature is stored. I want to learn more about the feature itself.

Well that's probably a bad example as we do include MDN links in CSS Wrapped, as well as explaining each feature on the site, and for learn more they may well want to know more about why it's showing as not supported when they think it is. 😁

But I agree in general "Learn more" isn't a great label for this link.

Still as per my last comment, I do think there's value in linking to the dist file because I see a LOT of confusion about why a feature it not marked as available in a certain browser when you think it should be, and it usually boils down to either 1) it not being available in the mobile version of that browser, 2) a certain feature not being supported (either due to bad data, or it just not being supported), and in the short term 3) data being out of date. So I still think a link someone to the dist file would be useful somewhere on the widget. Maybe smaller and with a different label.

As a short term solution I'm happy to update #75 to change the Learn More label to "Compatibility data" or a better solution, while you all work on the longer term solution of how to link to MDN. WDYT?

**Comment by captainbrosset** [comment]
> > Putting myself in the shoes of a web developer reading https://chrome.dev/css-wrapped-2024/ (which was mentioned previously), being interested in a feature, and clicking Learn more, I 100% agree that ending up on the feature's YML file (or the dist YML file for that matter) is super confusing and not useful. As a developer, I don't want to learn more about how and where the feature is stored. I want to learn more about the feature itself.
> 
> Well that's probably a bad example as we do include MDN links in CSS Wrapped, as well as explaining each feature on the site, and for learn more they may well want to know more about why it's showing as not supported when they think it is. 😁
> 
> But I agree in general "Learn more" isn't a great label for this link.

Fair enough. I guess I'm thinking about it from the angle of providing a component without knowing where this component will be embedded, and whether the hosting page will be explaining the feature.

> Still as per my last comment, I do think there's value in linking to the dist file because I see a LOT of confusion about why a feature it not marked as available in a certain browser when you think it should be, and it usually boils down to either 1) it not being available in the mobile version of that browser, 2) a certain feature not being supported (either due to bad data, or it just not being supported), and in the short term 3) data being out of date. So I still think a link someone to the dist file would be useful somewhere on the widget. Maybe smaller and with a different label.

Definitely smaller and with another label. Because I doubt that many developers will want to read and understand yaml dist files like https://github.com/web-platform-dx/web-features/blob/main/features/anchor-positioning.yml.dist

> As a short term solution I'm happy to update [#75](https://github.com/web-platform-dx/baseline-status/pull/75) to change the Learn More label to "Compatibility data" or a better solution, while you all work on the longer term solution of how to link to MDN. WDYT?

I'm good with this, if others are on board.

**Comment by bramus** [comment]
This came up _again_ at CSS Day 2026.

I _really_ think folks should make work of the proposed solution _(the second one)_ in https://github.com/web-platform-dx/baseline-status/issues/17#issuecomment-2694402305 here.

/cc @philipwalton

**Comment by tunetheweb** [comment]
#75 (change "Learn more" to "Compatibility data" and link to dist YML file) was approved 6 months ago but still not merged as no further feedback from any of the other maintainers 😔

It's not the solution we all want, but at least is more honest about what we have for now. Plus makes it possible (albeit a little crypticly) as to what individual things are missing from what platform to explain why something is not baseline when you may have thought it was.

**Comment by captainbrosset** [comment]
I'm fine merging that PR, as it's a good first step in the right direction. I originally didn't want to do this as I wanted others to share their opinions too. 
@devnook @atopal any feedback before we merge?

**Comment by devnook** [comment]
Let's merge
